### DISCUSSION TITLE
How can I replace the avatar with a user-defined image?

### DISCUSSION THREAD

**Discussion by ahsouri** [mearashadowfax/ScrewFast#242] [Q&A] [answered]
Hi,

I'm trying to change out some of avatars used in the testimonial with images I have in /public/professor.jpg. But I get this error: "Failed to parse URL from /professor.jpg". Am I allowed to use my own path or is it restricted to web-based sources?

**Comment by mearashadowfax** [comment, marked as answer]
You definitely can! If the image is in the public directory, you can use the normal `<img>` tag instead of the `Image` component.  Or check this discussion as well: https://github.com/mearashadowfax/ScrewFast/discussions/201
